_includes/footer.html

```diff
@@ -23,6 +23,10 @@ <h2 class="footer-heading">{{ site.title | escape }}</h2>
       <div class="footer-col social-links">
         {%- include social.html -%}
       </div>
+      <div class="footer-col signup-button">
+        {% include button.html url="{{ site.signup }}" %}
+        <a class="signup-button" href="{{ site.signup }}">signup</a>
```

Comment:
this would go inside the `_includes/button.html` file yeah? And since you're passing a url param, it would be 

something like 
```
{% assign url = include.url %}
<a class="signup-button" href="{{ url }}">signup</a>
```

Reply:
I am sorry, I read through all kinds of forums and tutorials for getting a stupid little button running, but none of the solutions seemed to work and I always got some syntax error. Since I don't have much time to do this, I would rather merge the sign up link for now, in order to have it up live and then work on the styling later. I will make another issue for that.